lib/Dialect/FIRRTL/Transforms/ExpandWhens.cpp

```diff
@@ -729,6 +729,12 @@ LogicalResult ModuleVisitor::checkInitialization() {
 
     // Get the op which defines the sink, and emit an error.
     FieldRef dest = std::get<0>(destAndConnect);
+
+    // zero-width wires do not need to be initialized
+    if (auto op = type_dyn_cast<FIRRTLBaseType>(dest.getValue().getType()))
+      if (op.getBitWidthOrSentinel() == 0)
+        continue;
```

Comment:
Comment nit (PR body/commit nit): this is really any zero-width _type_, not just wires. This does avoid initialization checks for ports, registers, etc., yes?

Mega-comment nit: Capitalization/end sentences with a period. 😉